internal/runbits/requirements/requirements.go

```diff
@@ -42,28 +43,29 @@ func (pv *PackageVersion) Set(arg string) error {
 	return nil
 }
 
-type primeable interface {
-	primer.Outputer
-	primer.Prompter
-	primer.Projecter
-	primer.Auther
-	primer.Configurer
-	primer.Analyticer
-	primer.SvcModeler
-}
+const latestVersion = "latest"
 
-type configurable interface {
-	keypairs.Configurable
+type RequirementOperationParams struct {
+	Output              output.Outputer
+	Prompt              prompt.Prompter
+	Project             *project.Project
+	Auth                *authentication.Auth
+	Config              *config.Instance
+	Analytics           analytics.Dispatcher
+	SvcModel            *model.SvcModel
+	RequirementName     string
+	RequirementVersion  string
+	RequirementBitWidth int
+	Operation           model.Operation
+	NsType              model.NamespaceType
```

Comment:
This is mixing different param types. Traditionally we use params for command input as well as struct instances passed down through the runner. 

Think of params as the things you send to a constructor. The rest should just remain arguments.

Apologies though; it didn't register to me that you are not working on a traditional runner but a runbit here. Perhaps it makes sense if you make this a struct with a constructor, which you then call `ExecuteRequirementOperation`.

I realize this is slightly getting into the refactor space. I don't intend for this change request to be more than a trivial amount of line changes. But I want to make sure we at least leave this code in a better place than we found it.